pkg/components/accesslog.go

```diff
@@ -65,11 +65,18 @@ func (c *loggingTransport) RoundTrip(r *http.Request) (*http.Response, error) {
 	var start = time.Now()
 	var resp, e = c.Wrapped.RoundTrip(r)
 	a.Duration = int(time.Since(start).Nanoseconds() / 1e6)
+	a.Status = resp.StatusCode
 	if e == nil {
-		a.Status = resp.StatusCode
 		a.HTTPContentType = resp.Header.Get("Content-Type")
+		if resp.StatusCode > 399 {
+			respData, err := io.ReadAll(resp.Body)
+			if err != nil {
+				runhttp.LoggerFromContext(r.Context()).Error(err)
+			}
+			a.Message = string(respData)
+		}
 	} else {
-		a.Status = transportd.ErrorToStatusCode(e)
```

Comment:
We shouldn't remove this line here. It basically maps come different kinds of errors to status code, more specifically timeouts from the client vs timeouts from the backend.

Reply:
Thank you. I have updated the pull request.